tests/functional/c/consider/consider_join2.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=missing-docstring,invalid-name,undefined-variable,multiple-statements
```

Comment:
I would name this file ``consider_join_for_non_empty_separator.py`` (can't suggest to rename file during review).

Reply:
Thank you, make sense. Renamed in an additional commit.